multy_core/src/golos/golos_transaction.cpp

```diff
@@ -303,8 +394,9 @@ class GolosTransactionDestination
     PropertyT<BigInt> amount;
 };
 
-GolosTransaction::GolosTransaction(BlockchainType blockchain_type)
-    : TransactionBase(blockchain_type),
+GolosTransaction::GolosTransaction(const Account& account)
+    : TransactionBase(account.get_blockchain_type()),
+      m_account(account),
```

Comment:
Account instance should not be locked for lifetime of transaction, please store the private key instead or add a transaction-level PropertyT for private key.